admin_console/forms/PartnerConfiguration.php

```diff
@@ -1089,7 +1096,10 @@ public function addAllDisplayGroups($permissionNames)
 									Kaltura_Client_SystemPartner_Enum_SystemPartnerLimitType::ACCESS_CONTROLS.'_max',
 									Kaltura_Client_SystemPartner_Enum_SystemPartnerLimitType::LIVE_STREAM_INPUTS.'_max',
 									Kaltura_Client_SystemPartner_Enum_SystemPartnerLimitType::LIVE_STREAM_OUTPUTS.'_max',
-									Kaltura_Client_SystemPartner_Enum_SystemPartnerLimitType::LIVE_RTC_STREAM_INPUTS.'_max')
+									Kaltura_Client_SystemPartner_Enum_SystemPartnerLimitType::LIVE_RTC_STREAM_INPUTS.'_max',
+									'partner_level_allowed_domains',
```

Comment:
why the following are under LIVE_RTC_STREAM_INPUTS?

Reply:
it was more visually pleasing